MAUI/DataForm/grouping.md

```diff
@@ -0,0 +1,386 @@
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+The order of the editors in the group can also be changed by handling the `GenerateDataFormItem` event using `RowOrder` property of a `DataFormItem`.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem.FieldName == "FirstName")
+            {
+                e.DataFormItem.RowOrder = 0;
+            }
+            else if (e.DataFormItem.FieldName == "LastName")
+            {
+                e.DataFormItem.RowOrder = 2;
+            }
+            else if (e.DataFormItem.FieldName == "MiddleName")
+            {
+                e.DataFormItem.RowOrder = 1;
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the editors order of the group in grid rows
+
+The order of the editors of group in grid row can be changed within a row by using `RowOrder` and `ItemsOrderInRow` properties[DataFormDisplayOptions] attribute and by handling `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight C# %}
+
+    [Display(GroupName = "Name")]
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 0)]
+    public string FirstName { get; set; }
+
+    [Display(GroupName = "Name")]
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 2)]
+    public string LastName { get; set; }
+
+    [Display(GroupName = "Name")]
+    [DataFormDisplayOptions(RowOrder = 0, ItemsOrderInRow = 1)]
+    public string MiddleName { get; set; }
+
+{% endhighlight %}
+{% highlight C# %}
+
+        if (e.DataFormGroupItem != null)
+        {
+            if (e.DataFormGroupItem.GroupName == "Name")
+            {
+                e.DataFormGroupItem.ColumnCount = 3;
+            }
+        }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the group name for the editors
+
+The `GroupName` for the group can be changed in the `GenerateDataFormItem` event.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+       private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null)
+        {
+            if (e.DataFormItem != null)
+            {
+                if (e.DataFormItem.GroupName == "Name")
+                {
+                    e.DataFormItem.GroupName = "Name Group";
+                }
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Changing the layout for the group
+
+The linear or grid layout for the particular group can be loaded by handling the `GenerateDataFormItem` event.
+By setting the `ColumnCount` property in the data form, non-grouped items only will be arranged in the grid layout. To load the grid layout, set the [ColumnCount] for the `DataFormGroupItem`
```

Comment:
By setting the `ColumnCount` property in the data form, non-grouped items only will be arranged in the grid layout. To load the grid layout, set the [ColumnCount] for the `DataFormGroupItem.`

Reply:
Content changed